en/asset/load-assets.md

```diff
@@ -0,0 +1,226 @@
+
+@ccclass("NewScript")
+export class NewScript extends Component {
+
+    @property({type: SpriteFrame})
+    public spriteFrame: SpriteFrame = null;
+}
+```
+
+## How to set Assets in the Property inspector
+
+As long as the type is defined in the script, you can easily set the __asset__ directly in the __Property inspector__. Suppose, we create a script like this:
+
+```typescript
+// NewScript.ts
+import { _decorator, Component, SpriteFrame, Texture2D } from "cc";
+const { ccclass, property } = _decorator;
+
+@ccclass("NewScript")
+export class NewScript extends Component {
+
+    @property({type: Texture2D})
+    public texture: Texture2D = null;
+
+    @property({type: SpriteFrame})
+    public spriteFrame: SpriteFrame = null;
+}
+```
+
+After adding it to a node, it looks like this in the **Property inspector**:
+
+![asset-in-properties-null](load-assets/asset-in-inspector-null.jpg)
+
+__Next__, we drag a __Texture__ and a __SpriteFrame__ from the **Explorer** panel into the corresponding properties of the **Property Inspector**:
+
+![asset-in-properties-dnd](load-assets/asset-in-inspector-dnd.jpg)
+
+Resulting in：
+
+![asset-in-properties-dnd](load-assets/asset-in-inspector.jpg)
+
+This allws you to get and set assets directly within a script:
+
+```typescript
+start () {
+    const texture = this.texture;
+    const spriteFrame = this.spriteFrame;
+}
+```
+
+Although it is intuitive to set assets in the **Attributes Inspector**, the assets can only be set in the __Scene__, in advance. This means there is no way to switch assets dynamically. If you need to switch assets dynamically, take a look at this next section on __Dynamic loading__.
+
+## Dynamic loading
+
+There are two points to note when loading assets dynamically. 
+
+__First__, is that all assets that need to be dynamically loaded through a script must be placed in the `assets` folder or its subfolders. `assets` need to be created manually in the `assets` folder and must be located in the root directory of `assets/`, like this:
+
+![asset-in-properties-null](load-assets/assets-file-tree.jpg)
+
+> **Note:** The assets in the folder can refer to other assets outside the folder, and can also be referenced by external scenes or assets. When the project is built, except for the scenes that have been checked in the **Build** panel, all assets in the `assets` folder, together with the assets outside the `assets` folder that they are dependent on, will be deleted.
+
+> **Note:** If an asset is only dependent on other assets in assets and does not need to be called directly by `loader.loadRes`, then ** Please do not put it in the assets folder. Otherwise, the size of the package body and settings.js will be increased, and the useless assets in the project will not be automatically removed during the build process. At the same time, during the construction process, JSON's automatic merge strategy will also be affected, and it is not possible to merge fragmented JSON as much as possible.
```

Comment:
Please change all the `assets folder` to `resources folder`. Assets folder is where all projects assets are hold. `resources folder` is a specific folder, `assets/resources`, which hold all dynamic assets. Really need to keep the name here to make things clear